### ISSUE TITLE
plot_snrifar bug

### ISSUE THREAD

**Issue by duncan-brown** [ligo-cbc/pycbc#610]
```python
Traceback (most recent call last):
  File "/home/dbrown/src/pycbc-dev/bin/pycbc_page_snrifar", line 4, in 
    __import__('pkg_resources').run_script('PyCBC===e078f9', 'pycbc_page_snrifar')
  File "/home/dbrown/src/pycbc-dev/lib/python2.6/site-packages/pkg_resources/__init__.py", line 735, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/home/dbrown/src/pycbc-dev/lib/python2.6/site-packages/pkg_resources/__init__.py", line 1652, in run_script
    exec(code, namespace, namespace)
  File "/home/dbrown/src/pycbc-dev/lib/python2.6/site-packages/PyCBC-e078f9-py2.6.egg/EGG-INFO/scripts/pycbc_page_snrifar", line 123, in 
    plot_xmin = cstat_fore.min()
AttributeError: 'NoneType' object has no attribute 'min'
```

**Comment by cdcapano** [comment]
Oh, it looks like that is happening because there are no foreground triggers. Replace line 120 of pycbc_page_snrifar with:
```python
elif args.closed_box or cstat_fore.size == 0:
````

**Comment by cdcapano** [comment]
@duncan-brown You'll also have to replace line 130 with:
```python
if args.closed_box or cstat_fore.size == 0:
```

**Comment by cdcapano** [comment]
Check that... looks like cstat_fore is None if there is no foreground. So replace line 120 with:
```python
elif args.closed_box or cstat_fore is None:
```
and line 130 with:
```python
if args.closed_box or cstat_fore is None:
```

**Comment by ahnitz** [comment]
Collin,

I'm making a patch for this now. I have a file without foreground to test
on.

**Comment by cdcapano** [comment]
Also change line 184 to:
```python
 if args.not_cumulative and cstat_fore is not None:
```

**Comment by cdcapano** [comment]
Ok, thanks Alex.

**Comment by ahnitz** [comment]
Fixed by #612
